packages/design-system-tailwind-preset/src/typography.test.ts

```diff
@@ -93,6 +93,7 @@ describe('Typography', () => {
     const designTokens = await getDesignTokenVariables([
       '--typography',
       '--font-weight',
+      '--font-family',
```

Comment:
Adding `--font-family` variables to tests so it picks up the new `--font-family-sans`